ballerina-command/src/main/java/org/ballerinalang/command/cmd/UpdateCommand.java

```diff
@@ -86,13 +87,13 @@ public void setParentCmdParser(CommandLine parentCmdParser) {
 
     public static void update(PrintStream printStream) {
         String version = ToolUtil.getCurrentBallerinaVersion();
-        printStream.println("Fetching latest distribution version from remote server...");
+        printStream.println("Fetching the latest distribution version from the remote server...");
         String latestVersion = ToolUtil.getLatest(version, "patch");
         if (!latestVersion.equals(version)) {
             String distribution = ToolUtil.BALLERINA_TYPE + "-" + latestVersion;
             ToolUtil.downloadDistribution(printStream, distribution, ToolUtil.BALLERINA_TYPE, latestVersion);
             ToolUtil.useBallerinaVersion(printStream, distribution);
-            printStream.println("Updated to latest distribution version: " + latestVersion);
+            printStream.println("Updated to the latest Ballerina version: " + latestVersion);
```

Comment:
Shouldn't this be 

```suggestion
            printStream.println("Updated to the latest Ballerina version: '" + latestVersion + "'");
```

for consistency?